.github/workflows/nodejs.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [macOS-latest, ubuntu-latest, windows-latest]
```

Comment:
Windows failed during `npm run install:server` when it encountered the unresolved path `python3`.